package.json

```diff
@@ -7,6 +7,8 @@
   "homepage": "https://falling-fruit.vercel.app/",
   "dependencies": {
     "@fontsource/lato": "^4.3.0",
+    "@material-ui/core": "^4.12.3",
```

Comment:
Can we use the `X` icon from box-icons and get rid of this extra dependency?
e.g. `import { X } from '@styled-icons/boxicons-regular'`

We already have an icon library, so adding this could add bloat to our repo and make the interface inconsistent.